html/themes/custom/common_design_subtheme/templates/user/user.html.twig

```diff
@@ -0,0 +1,32 @@
+{#
+/**
+ * @file
+ * Theme override to present all user data.
+ *
+ * This template is used when viewing a registered user's page,
+ * e.g., example.com/user/123. 123 being the user's ID.
+ *
+ * Available variables:
+ * - content: A list of content items. Use 'content' to print all content, or
+ *   print a subset such as 'content.field_example'. Fields attached to a user
+ *   such as 'user_picture' are available as 'content.user_picture'.
+ * - attributes: HTML attributes for the container element.
+ * - user: A Drupal User entity.
+ *
+ * @see template_preprocess_user()
+ */
+#}
+
+{{ attach_library('common_design_subtheme/rw-user') }}
+
+<article{{ attributes.addClass('profile rw-profile') }}>
+
+
+  {% if logged_in %}
+    {% include '@common_design_subtheme/user/user-dashboard.html.twig' %}
+  {% endif %}
+
+  {% if content %}
+    {{- content -}}
+  {% endif %}
+</article>
```

Comment:
I think we could replace the content of this template with the content of `user-dashboard.html.twig` as that's all we want to display. What do you think?